src/utils/cli.ts

```diff
@@ -0,0 +1,28 @@
+
+type ValueFlags = '--glob';
+export function extractCliFlagValue(flag: ValueFlags): string | null {
+  const args = process.argv.slice(2);
```

Comment:
I know I did it originally, but should this util fn mutate process.argv?

I think we *need* to for some flags, as these will get passed to the next command and I think it might error if we don't.
But, we risk unintentionally removing flags that were intended for the next command? 

I think it would be better to have an explicit fn that removes these if we need to

Reply:
Right, so `umzug.runAsCLI()` will use `process.argv.slice(2)` as its args.
I don't think we should pass in args that are used both in our code and in the umzug code (at least by default), i.e. we should extract these values from argv by default. I have renamed the functions to make it more obvious.

Note that if we provide args to then they will overwrite the usage of `process.argv.slice(2)`, but I can't see why we would do this because then we would need to explicitly handle any umzug args and pass them through.